custom_components/tuya_local/devices/eeese_emil_dehumidifier.yaml

```diff
@@ -0,0 +1,187 @@
+name: Eeese Emil Dehumidifier
+products:
+  - id: DYD-S12A431 除湿机
+    name: Eeese Emil Dehumidifier
+primary_entity:
+  entity: humidifier
+  class: dehumidifier
+  name: Panel
```

Comment:
Correct. The reason I added it, is because without it, the entity is "Emil Dehumidifier None". 
<img width="344" alt="image" src="https://github.com/user-attachments/assets/d42486f6-adb7-49eb-a18b-412b190394d5">
Is there a way to remove that "None" ? Or is it fine to keep it ?

Reply:
This is a known issue (#1856 previously for switches only, but it seems it has spread to other entities).

But you can edit the name that is displayed, so it is not major.